app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/home/HomeScreen.kt

```diff
@@ -182,12 +182,12 @@ fun SurveyContent(
             },
             verticalArrangement = Arrangement.spacedBy(4.dp)
         ) {
-            SurveyText(
+            SurveyCrossfadeText(
```

Comment:
This change also seems out of scope for this PR 🤔

Reply:
I wanted to change the name of `SurveyBoldText` to `SurveyText` 😬 . So I needed to change this name to a different name.